pages/bla-report-repair.js

```diff
@@ -0,0 +1,62 @@
+import React, {useEffect, useState} from 'react';
```

Comment:
Is this file meant to be called `bla-report-repair.js`? What does the `bla` prefix for?